src/lib/viewers/office/__tests__/OfficeLoader-test.js

```diff
@@ -4,74 +4,107 @@ import * as file from '../../../file';
 import { PERMISSION_DOWNLOAD } from '../../../constants';
 
 const sandbox = sinon.sandbox.create();
-let fakeFile;
 
 describe('lib/viewers/office/OfficeLoader', () => {
-    beforeEach(() => {
-        fakeFile = {
-            extension: 'xlsx',
-            size: 1000,
-            permissions: {
-                can_download: true
-            },
-            representations: {
-                entries: [
-                    {
-                        representation: 'ORIGINAL'
-                    }
-                ]
-            }
-        };
-    });
-
     afterEach(() => {
         sandbox.verifyAndRestore();
     });
 
     describe('determineViewer()', () => {
-        it('should choose the Office viewer if it is not disabled and the file is ok', () => {
-            const viewer = OfficeLoader.determineViewer(fakeFile);
-            expect(viewer).to.deep.equal({
-                NAME: 'Office',
-                CONSTRUCTOR: OfficeViewer,
-                REP: 'ORIGINAL',
-                EXT: ['xlsx']
-            });
-        });
-
-        it('should choose the Office viewer if it is not disabled and the file is a shared link that is not password-protected', () => {
-            fakeFile.shared_link = {
-                is_password_enabled: false
-            };
+        const fakeFiles = [
+            {
+                extension: 'xlsx',
+                size: 1000,
+                permissions: {
+                    can_download: true
+                },
+                representations: {
+                    entries: [
+                        {
+                            representation: 'ORIGINAL'
+                        }
+                    ]
+                }
+            },
+            {
+                extension: 'xlsm',
```

Comment:
If you want to avoid repeating this block, you could keep the original fake file and then override the extension in the list here.
```
fakeFiles = [
{...fakeFile, extension: 'xlsx'},
{...fakeFile, extension: 'xlsb'},
{...fakeFile, extension: 'xlsm'},
];
```